### DISCUSSION TITLE
Can't view logs for any action steps

### DISCUSSION THREAD

**Discussion by josiahhusk-tomtom** [community/community#55165] [Actions]
### Select Topic Area

Bug

### Body

For all workflows in all of my projects, when I attempt to expand a specific step of a job to see the logs, the only thing I see is "Error: " in red. It doesn't matter if the step failed or succeeded. I have tried this in Chrome, Safari, and Firefox. When I ask other members of my development team to look at the same workflows and action steps, they see the logs just fine. Attaching a screenshot. Can anyone assist please?
<img width="1438" alt="Screenshot 2023-05-11 at 8 00 27 AM" src="https://github.com/community/community/assets/98489415/44d554f0-aaf8-4402-81cf-0b218c8c544d">

**Comment by dgrassellyb** [comment]
I have exactly the same problem since yesterday .. and some other teammates too ! 
@josiahhusk-tomtom have you still this problem or did you manage to solve it ?

**Reply by arunaru5** [reply]
I also faced the same issue , i resolved it by using below settings in browser:

In chrome browser settings->privacy and security ->site settings-> in recent activity find your github activity and allow local network access . it will fix the issue.

**Comment by harsha-chamarthi** [comment]
I am having the same error too, were you able to resolve it?

**Comment by ericis** [comment]
> [!TIP]
> If you know how to, check the browser's console logs and "Network" tab for errors (usually under "Developer Tools"). The GitHub Action logs use Azure storage URLs under the primary domain URL path `*.blob.core.windows.net/action-results/` that could be blocked by the network router or proxies between your browser connection and the internet.

**Reply by cr1tbit** [reply]
It was DNS. :)

using 8.8.8.8 fixed the issue for me.

**Reply by acaifara** [reply]
yup it was dns for me as well, set it up to 8.8.8.8 instead of my shitty provider (play poland)

**Reply by mihitha** [reply]
Thanks for the solution. That was Cloudflare ZeroTrust in my case !

**Comment by kolonuk** [comment]
This was caused by my VPN to my Azure network being connected. Disconnecting my VPN meant I could access public Azure IPs. Something screwy with routing.

**Comment by furqannick** [comment]
If you’re unable to view logs for any action steps, it’s usually due to one of these causes:

The workflow is still in queue or hasn’t started running yet.

The run is skipped due to conditions (e.g., if: filters).

You don’t have the required repository permissions to view logs.

The repo has private logs restricted by the org’s policy.

The workflow run was auto-cancelled (common with concurrent runs).

Try re-running the workflow, make sure you have at least Read access to Actions, and check if the workflow actually triggered (under Actions → Run history). If logs still don’t appear, an org admin may need to update permissions or security policies.

**Comment by AriESQ** [comment]
My adblocker was blocking a microsoft CDN domain; which resulted in these red ERROR: in job logs.

Pls close this issue if it is stale.

**Comment by JanKanis** [comment]
Here also the company VPN that routes several Azure domains. Configuring github in the browser to allow local network access fixed it without having to disable the VPN.
